### DISCUSSION TITLE
Bundle.css Help

### DISCUSSION THREAD

**Discussion by illusive-ch** [uicrooks/shopify-theme-lab#110] [Q&A]
Maybe I am thinking about this the wrong way but in the docs it says:

```During development when the start task is running, the browser console throws a bundle.css missing error.```

But when I run npm run start and use the development URL it loads up but none of the css is working because it is missing the bundle, so development on this URL seems pointless?

**Comment by sergejcodes** [comment]
The CSS should be loaded through JavaScript during development, it doesn't create a CSS bundle file.

**Comment by sergejcodes** [comment]
The latest `4.1.0` release now always creates a CSS bundle even during development.
